app/domain/etl/edition/metadata/number_of_files.rb

```diff
@@ -5,12 +5,12 @@ def self.number_of_files(body, extensions_regex)
     filter_links all_links, extensions_regex
   end
 
-  ALL_LINKS_XPATH = "//*[contains(@class, 'attachment-details')]//a/@href".freeze
+  ALL_LINKS_XPATH = "//*[contains(@class, 'attachment-details') or contains(@class, 'form-download')]//a/@href".freeze
```

Comment:
Can you create Migration to fix existing data?

Reply:
Now changed to a rake task.